### ISSUE TITLE
Create Overpass QL contains non-ascii chracters...

### ISSUE THREAD

**Issue by y-kim** [maproulette/maproulette2#310]
It isn't available to create an Overpass QL which contains non-ascii characters.
Strictly, subset of Unicode characters are allowed in Overpass QL (U+0020~U+0072, U+000A). All other characters are stripped by Javascript statement in challengeForm.scala.html:99.

**Comment by mvexel** [comment]
Good point, it would be good to address this. Anyone want to investigate and supply a PR?

**Comment by robybully** [comment]
This issue belongs maybe also to special characters in different languages.
The special characters in Overpass API textfield are lost and the query delivers no results although the challenge is built correct and Completed.

For example:
["wikipedia"~"de:Liste der Kulturdenkmäler in Bremen-Schwachhausen"] is saved as
["wikipedia"~"de:Liste der Kulturdenkmler in Bremen-Schwachhausen"] which delivers no result.
German special characters can be used: ÄÖÜäöüß...
If I change it to ["wikipedia"~"ler in Bremen-Schwachhausen"] it works.
